content/_code-samples/use-tickets/py/use-tickets-multisig.py

```diff
@@ -0,0 +1,132 @@
+from xrpl.models.transactions import TicketCreate, AccountSet, SignerListSet, SignerEntry
+from xrpl.transaction import safe_sign_and_submit_transaction, autofill, multisign, sign
+from xrpl.models.requests.account_objects import AccountObjects, AccountObjectType
+from xrpl.models.requests.submit_multisigned import SubmitMultisigned
+from xrpl.wallet import generate_faucet_wallet
+from xrpl.clients import JsonRpcClient
+
+# This code sample lists which tickets are outstanding against an account
+# and use the ticket for a multisignature transaction.
+# A funded account on the testnet is provided for testing purposes
+# https://xrpl.org/tickets.html#tickets
```

Comment:
This page has good explanation on how users can use Tickets to collect signatures for multisigning. Probably can add it as a reference:
https://xrpl.org/use-tickets.html

Reply:
Done, thanks :)